mtools/mloginfo/sections/clients_section.py

```diff
@@ -0,0 +1,97 @@
+import re
+from collections import defaultdict
+
+from .base_section import BaseSection
+
+try:
+    from mtools.util.profile_collection import ProfileCollection
+except ImportError:
+    ProfileCollection = None
+
+
+class ClientSection(BaseSection):
+    """
+    ClientSection class.
+
+    This section goes through the logfile and extracts driver information from
+    connected clients.
+    """
+
+    name = "client driver info"
```

Comment:
The name should match the section (this should be `clients`)

Reply:
Corrected